Day-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,27 @@
 import React from 'react';
 import './Button.css';
+
+const NumberButton = (props) => {
+
+    return (
+        <button className={props.buttonStyle} 
+                onClick={ 
+                    props.name === 'equals' ?
+                        () => props.onClick(props.operand1, props.operand2, props.aO)
+                    :
+                    
+                    props.op ? () => props.onClick(props.op) : ()=> props.onClick(props.text)
```

Comment:
keep your use of ternary operators consistent, this line has a different formatting from the ternary operator on the prior few lines